render/filters.go

```diff
@@ -244,6 +244,30 @@ func IsApplication(n report.Node) bool {
 // IsSystem checks if the node is a "system" node
 var IsSystem = Complement(IsApplication)
 
+// HasLabel checks if the node has the desired docker label
+func HasLabel(labelKey string, labelValue string) FilterFunc {
+	return func(n report.Node) bool {
+		value, _ := n.Latest.Lookup(docker.LabelPrefix + labelKey)
+
+		if value == labelValue {
+			return true
+		}
+		return false
+	}
+}
+
+// DoesNotHaveLabel checks if the node does NOT have the specified docker label, allowing for a more complete user-customizable filtering feature
+func DoesNotHaveLabel(labelKey string, labelValue string) FilterFunc {
```

Comment:
I would have done that, but I wasn't sure if that works the same way as when it was used with `IsSystem`, since this function doesn't simply return a bool